src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.5
```

Comment:
The SDK changes are not released yet so this line should be added to the 0.2.4 version